rpc/common/moc_common_storageinfo.proto

```diff
@@ -11,6 +11,7 @@ import "moc_common_common.proto";
 message StorageContainerInfo {
 	Capacity capacity  = 1;
 	Latency latency	= 2;
+	string preferredOwner = 3;
```

Comment:
Can you provide more details on the property? Is this node/zone?

Reply:
It is 'preferred Owner node'. I will change it accordingly after further disucssion.

Reply:
What is the intent of this property.

Is this input or readonly?

Reply:
Yes, this is a read only field for user and populated by MOC logic.

Similar to fields like capacity, this info is filled in node agent layer from Failover Cluster and then sent to cloud agent in the Create/Get response. 

Based on feedback from Piyush, I am also adding zone name. However, for the best IO performance, we still need to pick a container that is active in the same node (and zone) if available.  So additional field of 'node name'(preferred owner) is also required.

Workflow
1. ECE action plan creates workload CSVs (one per node). Sets Affinity for each CSV to a Zone.  For CSVs in a particular zone, the preferred owners are changed such that each node is a preferred owner to a specific CSV.
2.  When container is created, preferred owner info is fetched from backing CSV in node agent and sent to cloud agent.
3. Node agent is not zone aware, so only node info is sent to cloud agent.
4. In cloud agent during VM creation, this node name / preferred owner field is to used to pick a container that is active same node or zone as VM for better IO performance.